NEXT_RELEASE.md

```diff
@@ -0,0 +1,370 @@
+- **Numbers determine integration order**: bc-1 → bc-2 → bc-5 → bc-10
+- **Avoid conflicts**: Check for existing branches with the same number
+- **Keep under 50** (higher numbers indicate too many breaking changes)
+
+## 🚀 Creating a Breaking Change Branch
+
+### Step 1: Create the Branch
+```bash
+# Start from the latest develop
+git checkout develop
+git pull origin develop
+
+# Create your breaking change branch
+git checkout -b bc-3-remove-legacy-widget develop
+```
+
+### Step 2: Make Your Changes
+Implement your breaking changes in the branch:
+```bash
+# Make your changes
+# Edit files, remove deprecated code, etc.
+
+# Stage changes
+git add .
+```
+
+### Step 3: Commit with Proper Format
+Use conventional commits with breaking change indicators:
+```bash
+git commit -m "feat!: remove legacy Widget class
+
+BREAKING CHANGE: Removed the deprecated Widget.legacy() method
+
+Breaking-Change-Category: API
+Breaking-Change-Impact: High
+Breaking-Change-Migration: Replace Widget.legacy() with Widget.current()
+Breaking-Change-Affects: All applications using Widget.legacy()
+
+The old Widget.legacy() method has been removed as part of the v3.0
+modernization. Applications should migrate to Widget.current() which
+provides the same functionality with improved performance."
+```
+
+### Step 4: Push the Branch
+```bash
+git push origin bc-3-remove-legacy-widget
+```
+
+## 📝 Commit Message Format
+
+### Basic Structure
+```
+<type>!: <description>
+
+BREAKING CHANGE: <summary>
+
+Breaking-Change-Category: <category>
+Breaking-Change-Impact: <impact-level>
+Breaking-Change-Migration: <migration-instructions>
+Breaking-Change-Affects: <affected-components>
+
+[Optional detailed explanation]
+```
+
+### Required Elements
+
+#### 1. Commit Type with Breaking Change Indicator
+- Use `feat!:` for new features that break compatibility
+- Use `fix!:` for fixes that break compatibility
+- Use `refactor!:` for refactors that break compatibility
+- The `!` after the type indicates a breaking change
+
+#### 2. BREAKING CHANGE Footer
+```
+BREAKING CHANGE: <brief description of what breaks>
+```
+
+#### 3. Structured Metadata
+```
+Breaking-Change-Category: API|Interface|Types|Behavior|Dependencies
+Breaking-Change-Impact: Low|Medium|High|Critical
+Breaking-Change-Migration: <clear migration instructions>
+Breaking-Change-Affects: <what components/users are affected>
+```
+
+### Metadata Guidelines
+
+#### Categories
+- **API**: Public API changes (method signatures, new/removed methods)
+- **Interface**: TypeScript interface changes
+- **Types**: Type definition changes
+- **Behavior**: Changed runtime behavior
+- **Dependencies**: Dependency changes affecting consumers
+
+#### Impact Levels
+- **Low**: Affects edge cases or rarely used features
+- **Medium**: Affects common use cases, straightforward migration
+- **High**: Affects core functionality, requires code changes
+- **Critical**: Major architectural changes, complex migration
```

Comment:
I think a breaking change is a breaking change. The impact level is too subjective to be useful. I'd prefer not having them.